tests/test_bin_builder.py

```diff
@@ -0,0 +1,178 @@
+from unittest import TestCase
+from unittest.mock import Mock, patch
+from roulette import Outcome, Wheel, BinBuilder
+
+
+class TestBinBuilder(TestCase):
+    def setUp(self):
+        self.bin_builder = BinBuilder()
+        self.wheel = Wheel()
+
+    def test_bins_are_filled_for_straight_bets(self):
+        straight_bet_odds = 35
+        straight_outcomes = {
+            0: Outcome("0", straight_bet_odds),
+            1: Outcome("1", straight_bet_odds),
+            36: Outcome("36", straight_bet_odds),
+            37: Outcome("00", straight_bet_odds)
+            }
```

Comment:
Instead of always testing against fixed bin ids, a better test strategy is to test against randomly generated bins along with special bins (e.g 0, 37)

Reply:
the existing test is fine. just wanted to share better test strategy.